control/state.py

```diff
@@ -162,6 +163,8 @@ class OmapGatewayState(GatewayState):
     """
 
     OMAP_VERSION_KEY = "omap_version"
+    OMAP_FILE_LOCK_NAME = "omap_file_lock"
```

Comment:
nit: Not sure about "FILE" in the variable name,  the argument is described as "name of the lock" by the docs.